src/components/Collapsable/docs.md

```diff
@@ -0,0 +1,28 @@
+A `<Collapsable />` wraps content that should be collapsed by default and can be expanded on click.
+
+Supported props:
+- `collapsable`: Whether the content should be collapsable.
```

Comment:
rm this prop, when using the component a text is always collapsable, if it shouldn't be the component wouldn't be used